src/NuGetTrends.Scheduler/NuGetSearchService.cs

```diff
@@ -42,11 +42,19 @@ public async Task<IPackageSearchMetadata> GetPackage(string packageId, Cancellat
             try
             {
                 // Search doesn't return matching id as the first result. MySqlConnector was the 7th, for example.
-                var package = (await _packageSearchResource.SearchAsync(packageId, SearchFilter, 0, 10, NugetLogger, token)).FirstOrDefault(p => p.Identity?.Id == packageId);
+                var package = (await _packageSearchResource.SearchAsync(packageId, SearchFilter, 0, 1, NugetLogger, token))
```

Comment:
FYI, you can craft a query like `packageid:mysqlconnector` to do this. This only works on nuget.org and isn't part of the official NuGet protocol.

Reply:
@loic-sharma that's interesting! Any examples I can take a look?
Maybe we try to hit this one first, if we get a 404 we fallback to search.

Reply:
Example: https://api-v2v3search-0.nuget.org/query?q=packageid:mysqlconnector

You shouldn't need to fallback to search. If this query doesn't return the expected package, the package either doesn't exist or has been unlisted.

Reply:
Oh that's straight forward. Thank you!

I thought of 'fallback' in case nuget-trends was running against nuget other than nuget.org but doubt that'd ever happen :)

Thanks for the review

Reply:
I'll merge this using the search for now. Since it's using all that API from `NuGet.Protocol` and I don't want to just edit in-place to hit that query endpoint. A followup PR will come to refactor this later.

Reply:
Opened #64